### PR TITLE
Fix insert extension test for local dev testing mode

### LINKED ISSUE #3363: test-insert-extension is failing locally
screen shot 2016-05-27 at 10 13 17 am

### PR DESCRIPTION
**Pull request by muxin**
Fixes #3363

### CODE REVIEW

File: test/functional/test-insert-extension.js [hunk 1]
```diff
@@ -178,7 +178,11 @@ describe('test-insert-extension', () => {
       expect(getMode().localDev).to.be.true;
       const script = calculateExtensionScriptUrl('examples.build/ads.amp.html',
           'amp-ad', true);
-      expect(script).to.equal('/base/dist/v0/amp-ad-0.1.js');
+      if (window.ampTestRuntimeConfig.useCompiledJs) {
```
Comment (zhouyx) [inline test/functional/test-insert-extension.js]: I feel in a test we should expect only one result. Is there a way to set `window.ampTestRuntimeConfig.useCompiledJs` and then create two tests to check this condition?

Comment (muxin) [inline test/functional/test-insert-extension.js]: Yeah, you're right. I will create two tests for this.

**Comment by muxin** [comment]
PTAL

File: src/insert-extension.js [hunk 2]
```diff
@@ -96,10 +96,11 @@ function isAmpExtensionScriptRequired(win, element, extension) {
  * @param {bool=} isTest
  * @return {string}
```
Comment (zhouyx) [inline src/insert-extension.js]: Please update comment.

Comment (muxin) [inline src/insert-extension.js]: Done

File: src/insert-extension.js [hunk 3]
```diff
@@ -93,13 +93,15 @@ function isAmpExtensionScriptRequired(win, element, extension) {
  * @visibleForTesting
  * @param {string} path Location path of the window
  * @param {string} extension
- * @param {bool=} isTest
+ * @param {boolean=} isTest
```
Comment (zhouyx) [inline src/insert-extension.js]: Oh! Thanks for fixing that ; )

**Comment by zhouyx** [comment]
LGTM

**Comment by cramforce** [comment]
LGTM

**Pull request merged**